### ISSUE TITLE
Combine navigation to sidebar

### ISSUE THREAD

**Issue by stephenway** [untangled-web/untangled-stylekit#4]
We need to combine our [parts collection](https://github.com/untangled-web/untangled-stylekit/blob/feature/new-ui/src/main/cards/guideui/ui.cljs#L189) navigation with each [part](https://github.com/untangled-web/untangled-stylekit/blob/feature/new-ui/src/main/cards/guideui/ui.cljs#L167) into a nested unordered list using something like the [navlist](https://github.com/untangled-web/untangled-stylekit/blob/feature/new-ui/src/main/cards/guideui/ui.cljs#L64).

Here's an overview of the layout we are needing, and main is where this combined nav approach will go.
![image](https://cloud.githubusercontent.com/assets/38894/21369534/418614b8-c6bc-11e6-9bb7-3f60965c1ea0.png)

**Comment by awkay** [comment]
So, I wrote the original code, and this is not specific enough that I would know what to do.

What goes in the App toolbar, View nav, and Main? 

I thought you were doing the layout...what help are you requesting, specifically?

**Comment by stephenway** [comment]
Yes, so this will give you a bit more detail on contents. I attempted at making this work, but I'm not able to combine the Parts and Part components into one navigation unfortunately.

![image](https://cloud.githubusercontent.com/assets/38894/21397221/cc4c0fda-c757-11e6-9a49-66d45a96138a.png)

I just need help getting the Parts and Part toolbars combined into an accordion so that we can have most of the navigation on the left, and the more details stuff on the right.
